components/button/style.scss

```diff
@@ -181,6 +181,15 @@
 		padding: 0 8px 1px;
 		font-size: 11px;
 	}
+
+	.components-external-link__icon {
```

Comment:
I wasn't entirely sure that these are the right selectors to use, or whether I should somehow specify a class of components-button__icon on the icon.

Reply:
The class should begin with `components-button__` since the node is in the Button component.